### ISSUE TITLE
[Feature Request]  A verification system please!

### ISSUE THREAD

**Issue by Anath4n** [BlueDevelopersInc/DiscordSRVUtils#133]
**Is your feature request related to a problem? Please describe.**
I think it would be a nice addition if the addon contains a verification system, this being widely used on many servers

**Describe the solution you'd like**
Create a verification system

**Describe alternatives you've considered**
Create a verification system x2

**Additional context**
If this is possible, I know a lot of us who use this plugin would like it!

**Comment by BlueTree242** [comment]
You did not explain what "verification system" means because it could be lots of things so clearify please

**Comment by LazyyPumpkinn** [comment]
Isn't the "link-account" feature of discordarv already kinda verification system? Like, you can disallow all permissions from @everyone role and make another role named something like @verified or @member and have the permissions allowed in that one?

You maybe meant to add a captcha verification system, but why would you need that in discordsrvutils or discordsrv, just get dedicated verification bot if you want captcha verification

**Comment by BlueTree242** [comment]
yeah, but the last part like the ticket system you could just use ticket bot :)... the purpose of this plugin is reduce the other bots in your server to make your server look better, Anyways if i don't hear from this issue in 24 hours it will be closed

**Comment by Anath4n** [comment]
Yep, I was referring to the captcha system, but you're right, I ended up using another bot that helped me with that, just adding more weight to discordsrv and not to much use.

**Comment by Anath4n** [comment]
Yep, I was referring to the captcha system, but you're right, I ended up using another bot that helped me with that, just adding more weight to discordsrv and not much use.
